controllers/flashsystemcluster_controller.go

```diff
@@ -679,6 +671,7 @@ func (r *FlashSystemClusterReconciler) ensureDefaultStorageClass(instance *odfv1
 			return err
 		}
 
+		// vered TODO: Why to delete SC with same name ??
```

Comment:
@tyichye19  - what do you think?

Reply:
We must delete the StorageClass as its parameters are different than what we expect, and we will create the correct StorageClass in the next Reconcile loop (the same as we did for deployment/service/serviceMonitor).
The other option is to fail, and wait until the user will delete this StorageClass manually - but I am not sure it's a good idea.

Reply:
Decided to fail reconcile in case SC with same name exists. 
Keep trying in loop. 
https://trello.com/c/Gjh8Qenp/160-default-sc-reconcile-fail-in-case-sc-with-same-name-exists